content/en/workflows/access.md

```diff
@@ -0,0 +1,134 @@
+- Name: `[WF]: <your-workflow-name>`
+- Email: the email address associated with your Datadog account
+
+### View service account details
+
+To view a service account's details:
+1. On your workflow canvas, click on the automated trigger.
+1. Next to the service account, click **View**.
+
+{{< img src="workflows/service-account-details1.png" alt="Service account details" style="width:60%;" >}}
+
+## Action credentials
+
+Because workflow [actions][4] connect with external software systems, you may need to authenticate your Datadog account to the corresponding integration. A workflow can run successfully only if every workflow action that requires authentication can verify the identity of your Datadog account.
+
+Workflow actions can be authenticated in two ways:
+- Credentials and permissions configured in the integration tile
+- Custom connection credentials
+
+### Integration tile credentials
+
+Credentials and account authentication that you set up in the following Datadog integration tiles automatically propagate to the corresponding actions in Workflows:
+- Jira
+- PagerDuty
+- Slack
+
+Configure the integration tiles by following instructions in [Datadog Integrations][5].
+
+### Connection credentials
+
+Use a custom [connection][6] to authenticate a [generic action][7] or any action for which the integration tile does not offer authentication.The workflow connection object allows you to provide credentials to connect to an integration. The parameters vary by integration.
+
+To connect to an arbitrary service, use the HTTP Connection type, and choose from two authentication options:
+- Token based authentication
+- A username and password combination
+
+### Add credentials to an action
+
+Whether you configured credentials in the integration tile or created a custom connection, Workflows makes your credentials available for each workflow action. To add credentials to an action, follow the steps below:
+
+1. Navigate to the [Workflows list][8].
+1. Select the workflow containing the action to which you need to add a credential. The workflow builder appears.
+1. In the workflow visualization, click the action to which you need to add a credential.
+1. The right side panel populates with the action details.
+1. Under the **Configure** tab, look for the **Authentication** dropdown. Choose your desired credential from the dropdown. Or, click **Add New** to create a new credential.
+1. Click **Save**.
+
+## Workflow permissions
+
+Use [role-based access control (RBAC)][9] to control access to your workflows and connections. To see the list of permissions that apply to workflows and connections, see [Datadog Role Permissions][10].
+
+### Restrict connection use
+
+Set permissions on each connection to limit modifications or restrict their use. The granular permissions include **Viewer**, **Resolver**, and **Editor**.
+
+Viewer
+: Can view
+
+Resolver
```

Comment:
I wish we had a better definition for "resolve." I've asked in Slack and we can add it later in a clarifying PR if necessary.